src/ast/parser.h

```diff
@@ -23,7 +23,6 @@ namespace parser
     if (!parser.load_grammar(grammar.data(), grammar.size()))
       exit(-1);
 
-    parser.enable_packrat_parsing();
```

Comment:
Why aren't we enabling packrat parsing?

Reply:
I don't think our grammar currently requires it in any case, but I'm fine with putting it back if you think it may be needed in the future.

Reply:
There are a few examples already that require arbitrary look-ahead, like lambda syntax (which looks like a tuple followed by a block). I think we should leave it on.

Reply:
Alright, I've put it back